### ISSUE TITLE
Crash on saving patch (Fedora Linux)

### ISSUE THREAD

**Issue by Zreimbach** [plugdata-team/plugdata#1633]
Hi! I use PlugData on my laptop with Fedora Linux 40 and I have stable crashes when I try to save the patch.
It can be saving a new patch or resaving existing patch, nevermind. However, the patch saves.

**Comment by timothyschoen** [comment]
Hi! What version of plugdata are you using?

**Comment by Zreimbach** [comment]
> Hi! What version of plugdata are you using?

I'm using plugdata 0.8.3.

**Comment by timothyschoen** [comment]
Could you try a nightly build from https://plugdata.org/download.html, so we can see if it still happens with the latest changes?

**Comment by Zreimbach** [comment]
> Could you try a nightly build from https://plugdata.org/download.html, so we can see if it still happens with the latest changes?

I tried a nightly build for Fedora 37 and 38. But I couldn't execute the binary file and I got this error:
`./plugdata: error while loading shared libraries: libGLU.so.1: cannot open shared object file: No such file or directory
`

**Comment by dromer** [comment]
The Fedora 37 and 38 builds should probably be deprecated since these versions are no longer maintained.
Better to move to 39+40.

**Comment by Zreimbach** [comment]
> The Fedora 37 and 38 builds should probably be deprecated since these versions are no longer maintained. Better to move to 39+40.

Yep! But I couldn't find on the download page a nightly builds for 39+40. There are only 37 and 38 versions.
